### PR TITLE
EM4x05/EM4x69 command rewrite and improvements

### PR DESCRIPTION
**Pull request by marshmellow42**
This cleans up and clarifies the lf em commands
IMPORTANT NOTICE: lf em command structure has changed!!!  lf em4x em4x...  is now lf em 4x...
also lf em readword and lf em writeword have been changed to lf em 4x05readword and lf em 4x05 writeword to clear up some confusion.

other lf em 4x05 commands have been added along with the full read back of what you read automatically.

some small improvements were also made to lf psk and lf fsk demods.

also added em4x05/em4x69 chip detection to lf search

### CODE REVIEW

**Comment by pwpiwi** [comment]
Would you mind adding a few words to CHANGELOG.md ?

**Comment by marshmellow42** [comment]
I did, do I need more details?

**Comment by pwpiwi** [comment]
About the new command structure?

File: CHANGELOG.md [hunk 1]
```diff
@@ -38,6 +41,10 @@ This project uses the changelog in accordance with [keepchangelog](http://keepac
 - Added option c to 'hf list' (mark CRC bytes) (piwi)
 
 ### Changed
+- small changes to lf psk and fsk demods to improve results when the trace begins with noise or the chip isn't broadcasting yet (marshmellow)
+- NOTE CHANGED ALL `lf em4x em*` cmds to simpler `lf em ` - example: `lf em4x em410xdemod` is now `lf em 410xdemod`
+- Renamed and rebuilt `lf em readword` && readwordpwd to `lf em 4x05readword` - it now demods and outputs the read block (marshmellow/iceman)
```
Comment (marshmellow42) [inline CHANGELOG.md]: is this enough?

Comment (pwpiwi) [inline CHANGELOG.md]: Oh yes. :)

**Pull request merged**